jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/CaseRoleItemView.java

```diff
@@ -84,16 +238,40 @@ public void addAction(final CaseRolesPresenter.CaseRoleAction action) {
         a.setOnclick(e -> action.execute());
 
         final HTMLElement li = getDocument().createElement("li");
+        if (!action.isEnabled()) {
+            addCSSClass(li, "disabled");
```

Comment:
Although disabled, the action is still clickable, after which **Remove assignment** dialog gets opened.